src/lib.rs

```diff
@@ -42,6 +45,33 @@ use rocksdb_store::{self as backend};
 // todo: Look into unifying these types?
 pub use backend::{GetError, SetError};
 
+enum Location<'a> {
+    DataDir(&'a PlatformDefault),
```

Comment:
Rename to PlatformDefault

Reply:
Once again, this is already done. Are you looking at old code?

Reply:
https://github.com/johanhelsing/bevy_pkv/pull/26/files

Reply:
> https://github.com/johanhelsing/bevy_pkv/pull/26/files

Yes, these have the changes you asked for.

Reply:
```suggestion
    PlatformDefault(&'a PlatformDefault),
```

Reply:
Okay, I think we where miscommunication. I understand what your asking now. I'll make the changes.